x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/job/results/AutodetectResult.java

```diff
@@ -127,7 +127,7 @@ public AutodetectResult(StreamInput in) throws IOException {
         } else {
             this.modelPlot = null;
         }
-        if (in.getVersion().onOrAfter(Version.V_8_0_0)) {
+        if (in.getVersion().onOrAfter(Version.V_7_9_0)) {
```

Comment:
Since it's just a single field inside the check I would completely remove the check from master.  This will have to be done in 7.last if it's not done now, but this one seems unlikely to cause confusion if removed now, so might as well be removed now.

Reply:
Done.